tubular/slack.py

```
@@ -30,19 +29,23 @@ def submit_slack_message(auth_token, channels, message):
     post_url = "{}{}".format(SLACK_API_URL, NOTIFICATION_POST)
     # to remove slack API warning
     headers = {
-        'Content-type': CONTENT_TYPE
+        AUTH_HEADER_FIELD: 'Bearer ' + auth_token
     }
     for channel in channels:
-        params = {
-            "token": auth_token,
+        arguments = {
             "channel": channel,
             "text": message
         }
         response = requests.post(post_url,
```

SHould we want to wrap this whole thing in a try catch in case the requests.post() call throws an exeception?

---

I'm leaning in the opposite direction. I'd like this to fail more loudly in the future rather than failing silently as it has been.